lumigator/python/mzai/sdk/tests/conftest.py

```diff
@@ -51,3 +58,38 @@ def json_data_jobs() -> Traversable:
 @pytest.fixture(scope="session")
 def json_data_job() -> Traversable:
     return json_data_path() / "job.json"
+
+
+@pytest.fixture(scope="session")
+def json_data_experiments() -> Traversable:
```

Comment:
given that we return a fully-qualified path, we might want to return a string or a Path object for simplicity, generally in Python it's idiomatic that file objects are either `Path` or `string` objects, depending on the use-case